soda_machine.rb

```diff
@@ -7,12 +8,43 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+     @sodas.length
   end
 
   def find_soda(soda_brand)
+    return @sodas.find{|soda| soda.brand==soda_brand}
+    nil
```

Comment:
this line of code will never be reached.  the line above always returns, regardless.  so you could safely delete this line here (line 16)